### ISSUE TITLE
Al salir jugadores del evento

### ISSUE THREAD

**Issue by Lucheti29** [AthenaEventEngine/AthenaCore#20]
Si el evento se queda con un sólo jugador o sólo queda un team, debería terminarse.

**Comment by fissban** [comment]
en realidad estaba pensando si dejarlos deslogear en medio de un evento o no...q opinas?

**Comment by Lucheti29** [comment]
En que te basás para decir si los dejás desloguear? Yo creo que siempre pueden desloguear (Administrador de tareas -> Matar L2).

**Comment by u3games** [comment]
Claro, aun que añadas que no puedan abandonar.. externamente pueden. Deberiamos de añadir lo que dice @luksdlt92, que cuando se marchen del evento y no se cumplan los requisitos minimos del evento actual, se termine y otorgue ganador si es un team o simplemente se finalice.

**Comment by ghost** [comment]
Continues to happen?.

**Comment by Lucheti29** [comment]
Continues

**Comment by u3games** [comment]
Auto check players ON in event, fix this.... Example, every 2 minutes.
